<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvancedOp</class>
 <widget class="QMainWindow" name="AdvancedOp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>859</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Zaawansowane funkcje</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font-family: Arial;
	font-size: 20px;
	color: white;
	text-align: center;
}
#centralwidget{
	background-color: #0ca678;
}
QPushButton{
	height: 100px;
	border: 2px solid white;
	border-radius: 12px;
	color: white;
	background-color: transparent;
}
QPushButton:hover{
background-color: white; 
color: #0ca678;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>411</width>
      <height>651</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btnDer">
       <property name="text">
        <string>Pochodne</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnInt">
       <property name="text">
        <string>Całki</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLim">
       <property name="text">
        <string>Granica ciągu</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnExt">
       <property name="text">
        <string>Ekstremum funkcji</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>431</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;Zaawansowane funkcje&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBack">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: none;
border-radius: 0;</string>
    </property>
    <property name="text">
     <string>Wróć</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
